### PR TITLE
Issues since v1.11 release

### PR DESCRIPTION
**Pull request by protich**
Bug fixes and enhancements for v1.11

### CODE REVIEW

File: include/class.forms.php [hunk 1]
```diff
@@ -2198,6 +2198,9 @@ function toString($value) {
         if (is_array($value))
             return '';
 
+        if (is_array($value))
+            return '';
+
```
Comment (Chefkeks) [inline include/class.forms.php]: Despite that I don't have any idea where this code is used, but does it make sense to `return '';` twice?

Comment (protich) [reply include/class.forms.php]: Most definitely unnecessary - fixed. Thanks

**Comment by aydreeihn** [comment]
I see one more continue 2 warning:

PHP Warning:  "continue" targeting switch is equivalent to "break". Did you mean to use "continue 2"? in /var/www/html/develop-next/osTicket/setup/test/tests/class.php_analyze.php on line 78

Should this one be changed as well or was it intentionally left out?

**Comment by protich** [comment]
@aydreeihn - good catch, I'll add it to the PR shortly.

**Pull request merged**